### ISSUE TITLE
use trained model to separate files

### ISSUE THREAD

**Issue by JackTyl3r** [open-mmlab/mmdetection#8647]
Hi, before we start, I know this question may be out of scope, but it's still something related to mmdetection, so come on, I'm getting to know the tool and I trained a model (autoassign) to recognize a certain type of animal (dog specifically) and I would like to use this model trained in a script to separate images into a folder with several images of the trained animal and garbage (everything that is not a dog), the problem is that I am at a loss when it comes to knowing when the animal was detected after making the inference, reading a little the argmax in the variable that received the result of the inference seemed to solve my problem since it always returns a fixed value when it recognizes the animal in the image, but it is not always, so I would like to know if there is another most reliable method of knowing when my model has recognized the animal in the image so I can separate it from the rest.

**Comment by muhd-umer** [comment]
Have you seen https://github.com/open-mmlab/mmdetection/issues/8548#issue-1336286734, it seems quite similar to your situation. 
As for having a reliable method, maybe add a parameter of `--threshold` in your separation script that puts the pictures of dogs in the folder if the `result > threshold`, and continues otherwise.
